core/tools/messagehandler/loggingcategorymodel.cpp

```diff
@@ -50,6 +48,7 @@ LoggingCategoryModel::LoggingCategoryModel(QObject *parent)
     Q_ASSERT(m_instance == nullptr);
     m_instance = this;
     m_previousFilter = QLoggingCategory::installFilter(categoryFilter);
+    connect(this, &LoggingCategoryModel::addCategorySignal, this, &LoggingCategoryModel::addCategorySlot, Qt::QueuedConnection);
```

Comment:
connect needs to happen before installing the filter otherwise we miss log categories on app start.

Reply:
done